backend/vxlan/device.go

```diff
@@ -124,13 +124,26 @@ func (dev *vxlanDevice) Configure(ipa ip.IP4Net, flannelnet ip.IP4Net) error {
 	return nil
 }
 
+func (dev *vxlanDevice) ConfigureIPv6(ipn ip.IP6Net, flannelnet ip.IP6Net) error {
+	if err := ip.EnsureV6AddressOnLink(ipn, flannelnet, dev.link); err != nil {
+		return fmt.Errorf("failed to ensure v6 address of interface %s: %w", dev.link.Attrs().Name, err)
+	}
+
+	if err := netlink.LinkSetUp(dev.link); err != nil {
+		return fmt.Errorf("failed to set v6 interface %s to UP state: %w", dev.link.Attrs().Name, err)
+	}
+
+	return nil
+}
+
 func (dev *vxlanDevice) MACAddr() net.HardwareAddr {
 	return dev.link.HardwareAddr
 }
 
 type neighbor struct {
 	MAC net.HardwareAddr
 	IP  ip.IP4
```

Comment:
Do we want to support IPv6-only installations? I would assume "yes" and in that case, I would convert this to a pointer as well.

Reply:
Still remains unanswered ;)

Reply:
Thanks for the heads up, I had forgot about this. Yes! Good idea. I must change this

Reply:
After looking a bit more into this, I think this change should be prepared in a separated PR because it requires a not small refactor. I created an issue about this => https://github.com/flannel-io/flannel/issues/1453

Reply:
OK, sounds good :+1: